packages/embedder/workflows/embedMetadata.ts

```diff
@@ -61,39 +63,40 @@ export const embedMetadata: JobQueueStepRun<
       forceBuildText: true
     })
   }
+  // Delete existing chunks for this metadataID to prevent stale data
+  await pg
+    // cast to any because these types won't be available in CI
+    .deleteFrom(embeddingModel.tableName as any)
```

Comment:
i know this comment came from of my old code, but we don't need to cast as any anymore. we do actually generate types for this table in CI.